clients/instance/ibm-pi-volume.go

```diff
@@ -215,6 +228,10 @@ func (f *IBMPIVolumeClient) UpdateVolumeAttach(id, volumeID string, body *models
 
 // Performs action on volume
 func (f *IBMPIVolumeClient) VolumeAction(id string, body *models.VolumeAction) error {
+	// Check for satellite differences in this endpoint
+	if strings.Contains(f.session.Options.Zone, helpers.PIStratosRegionPrefix) && body.ReplicationEnabled != nil {
```

Comment:
one small comment: Could we have a common function `isStratos()` which can be used everywhere?
```suggestion
	if body.ReplicationEnabled != nil && isStratos() {
```

Reply:
Made changes.